codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -34,19 +34,19 @@
 public class SpdyFrameCodec extends ByteToMessageDecoder
         implements SpdyFrameDecoderDelegate, ChannelOutboundHandler {
 
-    private static final SpdyProtocolException INVALID_FRAME =
-            new SpdyProtocolException("Received invalid frame");
+    protected static final SpdyProtocolException INVALID_FRAME =
+        new SpdyProtocolException("Received invalid frame");
 
-    private final SpdyFrameDecoder spdyFrameDecoder;
-    private final SpdyFrameEncoder spdyFrameEncoder;
+    protected final SpdyFrameDecoder spdyFrameDecoder;
+    protected final SpdyFrameEncoder spdyFrameEncoder;
     private final SpdyHeaderBlockDecoder spdyHeaderBlockDecoder;
     private final SpdyHeaderBlockEncoder spdyHeaderBlockEncoder;
 
     private SpdyHeadersFrame spdyHeadersFrame;
     private SpdySettingsFrame spdySettingsFrame;
 
-    private ChannelHandlerContext ctx;
-    private boolean read;
+    protected ChannelHandlerContext ctx;
+    protected boolean read;
```

Comment:
I am no a fan of have protected variables which are not final... Can we keep them private and just add protected methods that allow to get the values ?

Reply:
I need to override the `read` parameter in 
```java
    @Override
    public void readCustomFrame(final CustFrame custFrame) {
        read = true;
        ctx.fireChannelRead(custFrame);
    }
``` 
and the `readCustomFrame(...)` method presents on `ACCSSpdyFrameDecoderDelegate` which is something like
```java
public interface ACCSSpdyFrameDecoderDelegate extends SpdyFrameDecoderDelegate {

    void readCustomFrame(final CustFrame custFrame);
}
```